Solutions/NGINX HTTP Server/Data/Solution_Nginx.json

```diff
@@ -1,8 +1,8 @@
 {
-  "Name": "Nginx",
-  "Author": "Sanmit Biraj - v-sabiraj@microsoft.com",
-  "Description": "NGINX HTTP Server data connector provides the capability to ingest [NGINX HTTP Server](https://nginx.org/en/) events into Azure Sentinel. Refer to [NGINX Logs documentation](https://nginx.org/en/docs/http/ngx_http_log_module.html) for more information.",
-  "WorkbookDescription": "Sets the time name for analysis",
+  "Name": "NGINX HTTP Server",
+  "Author": "Microsoft - support@microsoft.com",
+   "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/Azure_Sentinel.svg\" width=\"75px\" height=\"75px\">",
+  "Description": "The [NGINX HTTP Server](https://nginx.org/) data connector provides the capability to ingest [NGINX HTTP Server](https://nginx.org/#basic_http_features) events into Microsoft Sentinel. Refer to [NGINX Logs documentation](https://nginx.org/en/docs/http/ngx_http_log_module.html) for more information. \n \n **Underlying Microsoft Technologies used:**\n\nThis solution takes a dependency on the following technologies, and some of these dependencies either may be in [Preview](https://azure.microsoft.com/support/legal/preview-supplemental-terms/) state or might result in additional ingestion or operational costs: \n \n a. [Azure Monitor HTTP Data Collector API](https://docs.microsoft.com/azure/azure-monitor/logs/data-collector-api)",
```

Comment:
hyperlink is only for `NGINX`. Please remove hyperlink for `HTTP Server` as it is not request as per text change document shown below.
![image](https://user-images.githubusercontent.com/107389644/175504782-0a6c72f0-aec6-4cc8-9294-4621fa67dfd1.png)